Not allowing commits due to eslint errors

Trying to do a commit in my fork of this repository will produce

```sh
$ git commit -m "Removes # from Stewards list. Closes #269" -a                                 
✔ Preparing lint-staged...              
⚠ Running tasks for staged files...                                                                                                                             
  ❯ package.json — 1 file                                                                                                                                       
    ❯ packages/app/**/*.{ts,tsx} — 1 file                                       
      ✖ yarn workspace @gooddollar/goodcollective-app lint --fix [FAILED]                                                                                       
      ◼ yarn workspace @gooddollar/goodcollective-app format                                                                                                    
    ↓ packages/sdk-js/**/*.{ts,tsx} — no files                                  
↓ Skipped because of errors from tasks.                                                                                                                         
✔ Reverting to original state because of errors...                                                                                                              
✔ Cleaning up temporary files...        
                                                                                                                                                                
✖ yarn workspace @gooddollar/goodcollective-app lint --fix:                                                                                                     

/home/vtamara/comp/gooddollar/GoodCollective/packages/app/src/components/DonateComponent.tsx
  58:31  warning  'type' is defined but never used. Allowed unused args must match /^_/u  @typescript-eslint/no-unused-vars

home/vtamara/comp/gooddollar/GoodCollective/packages/app/src/components/Header/ConnectWalletMenu.tsx                                                           
  7:35  warning  'props' is defined but never used. Allowed unused args must match /^_/u  @typescript-eslint/no-unused-vars                                     
                                                                                                                                                                
/home/vtamara/comp/gooddollar/GoodCollective/packages/app/src/components/TransactionList/TransactionList.tsx                                                    
  1:29  warning  'Platform' is defined but never used  @typescript-eslint/no-unused-vars                                                                      
                                        
/home/vtamara/comp/gooddollar/GoodCollective/packages/app/src/components/ViewCollective.tsx                                                                     
  56:26  warning  'isMobileView' is assigned a value but never used  @typescript-eslint/no-unused-vars                                                          
                                                                                
/home/vtamara/comp/gooddollar/GoodCollective/packages/app/src/components/WalletCards/DonorCollectiveCard.tsx                                                    
  30:3  warning  'isDesktopResolution' is defined but never used. Allowed unused args must match /^_/u  @typescript-eslint/no-unused-vars                       
                                                                                
/home/vtamara/comp/gooddollar/GoodCollective/packages/app/src/components/WalletCards/StewardCollectiveCard.tsx                                                  
  25:3  warning  'isDesktopResolution' is defined but never used. Allowed unused args must match /^_/u  @typescript-eslint/no-unused-vars                       
                                        
/home/vtamara/comp/gooddollar/GoodCollective/packages/app/src/hooks/useApproveSwapTokenCallback.ts                                                              
  40:6  error  React Hook useEffect has a missing dependency: 'refetch'. Either include it or remove the dependency array  react-hooks/exhaustive-deps          

✖ 7 problems (1 error, 6 warnings)                                                                                                                              
                                                                                                                                                                
husky - pre-commit hook exited with code 1 (error) 
```

--- a/packages/app/src/App.tsx
+++ b/packages/app/src/App.tsx
@@ -15,17 +15,17 @@ import ActivityLogPage from './pages/ActivityLogPage';
 import { Providers } from './Providers';
 import DonatePage from './pages/DonatePage';
 import { WagmiProvider } from 'wagmi';
-import { celo, mainnet, AppKitNetwork } from '@reown/appkit/networks'
+import { celo, mainnet, AppKitNetwork } from '@reown/appkit/networks';
 import { ApolloProvider } from '@apollo/client';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { createAppKit } from '@reown/appkit/react';
 import { Colors } from './utils/colors';
-import { WagmiAdapter } from '@reown/appkit-adapter-wagmi'
+import { WagmiAdapter } from '@reown/appkit-adapter-wagmi';
 import { useCreateSubgraphApolloClient, useCreateMongoDbApolloClient } from './hooks/apollo';
 import { MongoDbApolloProvider } from './components/providers/MongoDbApolloProvider';
 
 const queryClient = new QueryClient();
-const projectId = 'b1b7664bfba2f6ad5538aa7fa9a2404f'
+const projectId = 'b1b7664bfba2f6ad5538aa7fa9a2404f';
 const metadata = {
   name: 'GoodCollective',
   description: 'GoodCollective is a platform for creating and supporting social good projects.',
@@ -33,12 +33,12 @@ const metadata = {
   icons: ['https://avatars.githubusercontent.com/u/179229932'],
   redirect: {
     native: 'YOUR_APP_SCHEME://',
-    universal: 'https://www.gooddollar.org/'
-  }
-}
+    universal: 'https://www.gooddollar.org/',
+  },
+};
 
 const networks: [AppKitNetwork, AppKitNetwork] = [mainnet, celo];
-const wagmiAdapter = new WagmiAdapter({ networks, projectId })
+const wagmiAdapter = new WagmiAdapter({ networks, projectId });
 
 createAppKit({
   adapters: [wagmiAdapter],
@@ -46,7 +46,7 @@ createAppKit({
   projectId,
   metadata,
   defaultNetwork: celo,
-})
+});
 
 function App(): JSX.Element {
   const subgraphApolloClient = useCreateSubgraphApolloClient();
@@ -59,28 +59,28 @@ function App(): JSX.Element {
   return (
     <Providers>
       <WagmiProvider config={wagmiAdapter.wagmiConfig}>
-      <QueryClientProvider client={queryClient}>
-        <ApolloProvider client={subgraphApolloClient}>
-          <MongoDbApolloProvider client={mongoDbApolloClient}>
-            <SafeAreaView style={styles.body}>
-              <Routing.Router>
-                <Routing.Routes>
-                  <Routing.Route path="/" element={<HomePage />} />
-                  <Routing.Route path="/about" element={<AboutPage />} />
-                  <Routing.Route path="/collective/:id" element={<ViewCollectivePage />} />
-                  <Routing.Route path="/collective/:id/stewards" element={<ViewStewardsPage />} />
-                  <Routing.Route path="/collective/:id/donors" element={<ViewDonorsPage />} />
-                  <Routing.Route path="/profile/:id" element={<WalletProfilePage />} />
-                  <Routing.Route path="/profile/:id/activity" element={<ActivityLogPage />} />
-                  <Routing.Route path="/profile/" element={<WalletProfilePage />} />
-                  <Routing.Route path="/donate/:id" element={<DonatePage />} />
-                  <Routing.Route path="/privacy/" element={<PrivacyPage />} />
-                  <Routing.Route path="/tandc/" element={<TermsPage />} />
-                </Routing.Routes>
-              </Routing.Router>
-            </SafeAreaView>
-          </MongoDbApolloProvider>
-        </ApolloProvider>
+        <QueryClientProvider client={queryClient}>
+          <ApolloProvider client={subgraphApolloClient}>
+            <MongoDbApolloProvider client={mongoDbApolloClient}>
+              <SafeAreaView style={styles.body}>
+                <Routing.Router>
+                  <Routing.Routes>
+                    <Routing.Route path="/" element={<HomePage />} />
+                    <Routing.Route path="/about" element={<AboutPage />} />
+                    <Routing.Route path="/collective/:id" element={<ViewCollectivePage />} />
+                    <Routing.Route path="/collective/:id/stewards" element={<ViewStewardsPage />} />
+                    <Routing.Route path="/collective/:id/donors" element={<ViewDonorsPage />} />
+                    <Routing.Route path="/profile/:id" element={<WalletProfilePage />} />
+                    <Routing.Route path="/profile/:id/activity" element={<ActivityLogPage />} />
+                    <Routing.Route path="/profile/" element={<WalletProfilePage />} />
+                    <Routing.Route path="/donate/:id" element={<DonatePage />} />
+                    <Routing.Route path="/privacy/" element={<PrivacyPage />} />
+                    <Routing.Route path="/tandc/" element={<TermsPage />} />
+                  </Routing.Routes>
+                </Routing.Router>
+              </SafeAreaView>
+            </MongoDbApolloProvider>
+          </ApolloProvider>
         </QueryClientProvider>
       </WagmiProvider>
     </Providers>

--- a/packages/app/src/components/DonateComponent.tsx
+++ b/packages/app/src/components/DonateComponent.tsx
@@ -30,7 +30,7 @@ import FrequencySelector from './DonateFrequency';
 import NumberInput from './NumberInput';
 import { ApproveTokenImg, PhoneImg, StreamWarning, ThankYouImg } from '../assets';
 import { formatNumberWithCommas } from '../lib/formatFiatCurrency';
-type ConfigChainId = typeof config.chains[number]['id'];
+type ConfigChainId = (typeof config.chains)[number]['id'];
 
 interface DonateComponentProps {
   collective: Collective;
@@ -55,7 +55,7 @@ const WarningExplanation = ({ type }: any) => (
     </Text>
   </Text>
 );
-const NetworkExplanation = ({ type }: any) => (
+const NetworkExplanation = ({ _type }: any) => (
   <Text color="goodOrange.500">
     <Text>Switch in your wallet to the Celo network</Text>
   </Text>

--- a/packages/app/src/components/Header/ConnectWalletMenu.native.tsx
+++ b/packages/app/src/components/Header/ConnectWalletMenu.native.tsx
@@ -1,13 +1,13 @@
-import '@walletconnect/react-native-compat';  
+import '@walletconnect/react-native-compat';
 import { StyleSheet, TouchableOpacity, View } from 'react-native';
-import { AppKitButton } from '@reown/appkit-wagmi-react-native'
+import { AppKitButton } from '@reown/appkit-wagmi-react-native';
 
 export const ConnectWalletMenu = () => {
   return (
     <>
       <TouchableOpacity>
         <View style={styles.mobileButtonContentContainer}>
-        <AppKitButton />
+          <AppKitButton />
         </View>
       </TouchableOpacity>
     </>

--- a/packages/app/src/components/Header/ConnectWalletMenu.tsx
+++ b/packages/app/src/components/Header/ConnectWalletMenu.tsx
@@ -4,7 +4,7 @@ interface ConnectWalletMenuProps {
   dropdownOffset: { top: number; right?: number; left?: number };
 }
 
-export const ConnectWalletMenu = (props: ConnectWalletMenuProps) => {
+export const ConnectWalletMenu = (_props: ConnectWalletMenuProps) => {
   const { isDesktopView } = useScreenSize();
   return (
     <>

--- a/packages/app/src/components/Header/ConnectedAccountDisplay.tsx
+++ b/packages/app/src/components/Header/ConnectedAccountDisplay.tsx
@@ -30,7 +30,7 @@ export const ConnectedAccountDisplay = (props: ConnectedAccountDisplayProps) => 
 
   const handleNetworkClick = () => {
     if (isUnsupportedNetwork && switchChain) {
-      switchChain({chainId:SupportedNetwork.CELO});
+      switchChain({ chainId: SupportedNetwork.CELO });
       setChainName(formatChainName(chain?.name));
     }
   };
@@ -38,7 +38,7 @@ export const ConnectedAccountDisplay = (props: ConnectedAccountDisplayProps) => 
   useEffect(() => {
     if (isUnsupportedNetwork && switchChain) {
       setChainName(formatChainName(chain?.name));
-      switchChain({chainId:SupportedNetwork.CELO});
+      switchChain({ chainId: SupportedNetwork.CELO });
     }
   }, [isUnsupportedNetwork, switchChain, chain?.name]);
 

--- a/packages/app/src/components/TransactionList/SupportTransactionListItem.tsx
+++ b/packages/app/src/components/TransactionList/SupportTransactionListItem.tsx
@@ -1,6 +1,6 @@
 import { SupportTx } from '../../models/models';
 import { formatAddress } from '../../lib/formatAddress';
-import { useEnsName, useWaitForTransactionReceipt} from 'wagmi';
+import { useEnsName, useWaitForTransactionReceipt } from 'wagmi';
 import TransactionListItem from './TransactionListItem';
 import { useFetchFullName } from '../../hooks/useFetchFullName';
 import { Text } from 'react-native';

--- a/packages/app/src/components/TransactionList/TransactionList.tsx
+++ b/packages/app/src/components/TransactionList/TransactionList.tsx
@@ -1,4 +1,4 @@
-import { Image, Text, View, Platform, StyleSheet } from 'react-native';
+import { Image, Text, View, StyleSheet } from 'react-native';
 import { Link } from 'native-base';
 
 import { InterRegular, InterSemiBold } from '../../utils/webFonts';

--- a/packages/app/src/components/ViewCollective.tsx
+++ b/packages/app/src/components/ViewCollective.tsx
@@ -53,7 +53,7 @@ const HasDonatedCard = ({
 }) => {
   const { address } = useAccount();
   const { data: ensName } = useEnsName({ address, chainId: 1 });
-  const { isDesktopView /*, isMobileView*/ } = useScreenSize();
+  const { isDesktopView } = useScreenSize();
   const userName = ensName ?? 'This wallet';
   const { navigate } = useCrossNavigate();
   const isDonating = donorCollective && donorCollective.flowRate !== '0';

--- a/packages/app/src/components/ViewCollective.tsx
+++ b/packages/app/src/components/ViewCollective.tsx
@@ -53,7 +53,7 @@ const HasDonatedCard = ({
 }) => {
   const { address } = useAccount();
   const { data: ensName } = useEnsName({ address, chainId: 1 });
-  const { isDesktopView, isMobileView } = useScreenSize();
+  const { isDesktopView /*, isMobileView*/ } = useScreenSize();
   const userName = ensName ?? 'This wallet';
   const { navigate } = useCrossNavigate();
   const isDonating = donorCollective && donorCollective.flowRate !== '0';

--- a/packages/app/src/components/WalletCards/DonorCollectiveCard.tsx
+++ b/packages/app/src/components/WalletCards/DonorCollectiveCard.tsx
@@ -27,7 +27,7 @@ function DonorCollectiveCard({
   donorCollective,
   ensName,
   tokenPrice,
-  isDesktopResolution,
+  _isDesktopResolution,
 }: DonorCollectiveCardProps) {
   const { navigate } = useCrossNavigate();
   const userName = ensName ?? 'This wallet';

--- a/packages/app/src/components/WalletCards/StewardCollectiveCard.tsx
+++ b/packages/app/src/components/WalletCards/StewardCollectiveCard.tsx
@@ -22,7 +22,7 @@ function StewardCollectiveCard({
   collective,
   ensName,
   tokenPrice,
-  isDesktopResolution,
+  _isDesktopResolution,
 }: StewardCollectiveCardProps) {
   const { navigate } = useCrossNavigate();
   const userName = ensName ?? 'This wallet';

--- a/packages/app/src/config.ts
+++ b/packages/app/src/config.ts
@@ -1,9 +1,9 @@
-import { http, createConfig } from '@wagmi/core'
-import { celo } from '@wagmi/core/chains'
+import { http, createConfig } from '@wagmi/core';
+import { celo } from '@wagmi/core/chains';
 
-export const config  = createConfig({
-    chains: [celo],
-    transports: {
-      [celo.id]: http("https://rpc.ankr.com/celo"),
-    },
-  });
+export const config = createConfig({
+  chains: [celo],
+  transports: {
+    [celo.id]: http('https://rpc.ankr.com/celo'),
+  },
+});

--- a/packages/app/src/hooks/useApproveSwapTokenCallback.ts
+++ b/packages/app/src/hooks/useApproveSwapTokenCallback.ts
@@ -26,7 +26,7 @@ export function useApproveSwapTokenCallback(
     [decimalAmountIn, duration, tokenIn.decimals]
   );
 
-  const { data: blockNumber } = useBlockNumber({ watch: true })
+  const { data: blockNumber } = useBlockNumber({ watch: true });
   const { data: allowance = 0n, refetch } = useReadContract({
     chainId: chain?.id,
     address: tokenIn.address as `0x${string}`,
@@ -34,11 +34,11 @@ export function useApproveSwapTokenCallback(
     functionName: 'allowance',
     args: [address, collectiveAddress],
   });
-  
-  // eslint-disable-next-line react-hooks/exhaustive-deps
+
   useEffect(() => {
-    refetch()
-  }, [blockNumber])
+    refetch();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [blockNumber]);
 
   const { data } = useSimulateContract({
     chainId: chain?.id,
@@ -52,7 +52,7 @@ export function useApproveSwapTokenCallback(
   const { isPending, isSuccess, isError, writeContractAsync } = useWriteContract();
 
   const handleApproveToken =
-  writeContractAsync === undefined
+    writeContractAsync === undefined
       ? undefined
       : async () => {
           toggleApproveSwapModalVisible(true);

--- a/packages/app/src/hooks/useGetTokenBalance.ts
+++ b/packages/app/src/hooks/useGetTokenBalance.ts
@@ -3,7 +3,6 @@ import { useEffect, useState } from 'react';
 import { config } from './../config';
 import { getBalance } from 'wagmi/actions';
 
-
 export const useGetTokenBalance = (
   currencyAddress: string,
   accountAddress: `0x${string}` | undefined,
@@ -14,8 +13,7 @@ export const useGetTokenBalance = (
 
   useEffect(() => {
     if (!currencyAddress || !accountAddress || chainId !== SupportedNetwork.CELO) return;
-    getBalance(
-      config,{
+    getBalance(config, {
       address: accountAddress,
       chainId: chainId,
       token: currencyAddress as `0x${string}`,

